keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -51,9 +51,8 @@
 
 The kube-scheduler configuration API `kubescheduler.config.k8s.io` was in alpha
 for several releases. We graduated it to beta in 1.19 as `v1beta1`. We introduced
-`v1beta2` and `v1beta3` in 1.22 and 1.23 respectively. We'd like to graduate it
-to GA in 1.25 and mark `v1beta2` as deprecated to avoid breaking existing users,
-we'll remove the support of `v1beta2` in 1.26.
+`v1beta2` and `v1beta3` in 1.22 and 1.23 respectively. And it was graduated to GA
+in 1.25 as `v1`. Also in 1.25, we marked `v1beta2` as deprecated and removed it in 1.26. 
```

Comment:
We should mark v1beta3 as deprecated in 1.26 and start the clock.

As already stated, we should only remove v1beta2 in 1.28 and v1beta3 in 1.29

Reply:
refer https://github.com/kubernetes/kubernetes/pull/112257

Reply:
Thanks @alculquicondor 
I have sent a PR to mark v1beta3 as deprecated https://github.com/kubernetes/kubernetes/pull/112269

Reply:
@SataQiu please add this details about v1beta3 in the KEP

Reply:
@alculquicondor Thanks, Updated.